### PR TITLE
Translate to Brazilian Portuguese the Processes section of the basic training

### PR DESCRIPTION
**Pull request by laribritto**

### CODE REVIEW

File: docs/basic_training/processes.pt.md [hunk 1]
```diff
@@ -0,0 +1,950 @@
+---
+description: Basic Nextflow Training Workshop
+---
+
+# Processes
```
Comment (laribritto) [inline docs/basic_training/processes.pt.md:5]:
```suggestion
# Processos
```

File: docs/basic_training/processes.pt.md [hunk 2]
```diff
@@ -0,0 +1,950 @@
...
+description: Material de treinamento básico do Nextflow
+---
+
+# Processos
+
+No Nextflow, um processo (`process`) é a primitiva de computação básica para executar funções estrangeiras (ou seja, scripts personalizados ou ferramentas).
+
+A definição do processo começa com a palavra-chave `process`, seguida pelo nome do processo e, finalmente, o corpo do processo delimitado por chaves.
+
+O nome do process é comumente escrito em letras maiúsculas por convenção.
```
Comment (iaradsouza1) [inline docs/basic_training/processes.pt.md:11]: Process inside backticks: O nome do `process` ...

Or

O nome do processo (`process`)

File: docs/basic_training/processes.pt.md [hunk 3]
```diff
@@ -0,0 +1,950 @@
...
+4. Uma condicional booleana opcional para acionar a execução do processo
+5. O comando a ser executado
+
+## Script
+
+O bloco `script` é uma string que define o comando a ser executado pelo processo.
+
+Um processo pode executar apenas um bloco `script`. Deve ser a última instrução quando o processo contém declarações de entrada e saída.
+
+O bloco `script` pode ser uma string de uma ou várias linhas. A de várias linhas simplifica a escrita de scripts não triviais compostos por vários comandos abrangendo várias linhas. Por exemplo:
```
Comment (iaradsouza1) [inline docs/basic_training/processes.pt.md:72]: Isn't that redundant? Maybe just "O bloco de várias linha simplifica a execução de uma série de comandos."

File: docs/basic_training/processes.pt.md [hunk 4]
```diff
@@ -0,0 +1,950 @@
...
+          input:
+          path '*'
+
+          output:
+          path 'top_10_linhas'
+
+          script:
+          """
+          cat * > concatenado.txt
+          head -n 20 concatenado.txt > top_10_linhas
```
Comment (jvfe) [inline docs/basic_training/processes.pt.md:377]: If it's `head -n 20` shouldn't it be top_20_linhas?

**Review by jvfe — APPROVED** [review]

File: docs/basic_training/processes.pt.md (hunk 4 above)
Comment (mribeirodantas) [reply docs/basic_training/processes.pt.md:377]: Great point 😆 I have written down to fix this on the original material in English. Once fixed there, I will come to fix this here.

File: docs/basic_training/processes.pt.md (hunk 3 above)
Comment (mribeirodantas) [reply docs/basic_training/processes.pt.md:72]: I agree the writing isn't the best, but we shouldn't stray too far from the original material in English. I have written it down to improve in the original material and once fixed there, we can come back here.

File: docs/basic_training/processes.pt.md (hunk 2 above)
Comment (mribeirodantas) [reply docs/basic_training/processes.pt.md]: Thanks for spotting this one! 😄

**Pull request merged**